test/integration/__snapshots__/blocks-raw-handling.test.js.snap

```diff
@@ -80,7 +80,7 @@ exports[`Blocks raw handling pasteHandler wordpress 1`] = `"Howdy<br>This is a p
 
 exports[`Blocks raw handling should correctly handle quotes with mixed content 1`] = `
 "<!-- wp:quote -->
-<blockquote class="wp-block-quote"><!-- wp:heading {"level":1} -->
+<blockquote class="wp-block-quote"><!-- wp:heading {"level":1,"className":"wp-block-heading"} -->
```

Comment:
Not sure why you'd ever have content to convert that has such a class, but this is the test 🤷‍♀️ 

```js
const filtered = serialize(
	pasteHandler( {
		HTML: '<blockquote><h1 class="wp-block-heading">chicken</h1><p>ribs</p></blockquote>',
		mode: 'AUTO',
	} )
);	
```

Reply:
indeed weird class :)